tests/unified/runner.py

```
@@ -0,0 +1,528 @@
+                self.harness.output_threads.append(thread)
+
+            # Give services a moment to settle
+            await asyncio.sleep(2)
+
+            # 2. Load Tests
+            if self.test_file:
+                test_files = [self.test_file]
+            else:
+                if not self.tests_dir:
+                    raise ValueError("tests_dir must be set if test_file is not")
+                test_files = sorted(list(self.tests_dir.glob("*.json")))
```

Ruff (0.14.7)

Unnecessary `list()` call within `sorted()`

Remove the inner `list()` call

(C414)